guide/src/initialize_a_client.md

```diff
@@ -48,8 +48,10 @@ As it is usual with Rust, you must declare the dependency in your
 {{#include ../samples/Cargo.toml:secretmanager}}
 ```
 
-The default initialization is designed to meet the requirements for most
-cases. You create a default client using `new()`:
+Clients are created by calling `build()` on a
+[`ClientBuilder`][gax-client-builder]. The following creates a client with
+default configuration, which is designed to meet requirements for most use
```

Comment:
Consider:

```suggestion
Clients are created by calling `build()` on a
[`ClientBuilder`][gax-client-builder]. To obtain an appropriate `ClientBuilder` use
`Client::builder()`. The following creates a client with
default configuration, which is designed to meet requirements for most use
```

Reply:
Reworded using some of your suggestion. You might not like what I went with.